packages/unified-latex-to-pretext/libs/pre-conversion-subs/break-on-boundaries.ts

```diff
@@ -0,0 +1,134 @@
+import { env, arg } from "@unified-latex/unified-latex-builder";
+import * as Ast from "@unified-latex/unified-latex-types";
+import { getNamedArgsContent } from "@unified-latex/unified-latex-util-arguments";
+import {
+    anyEnvironment,
+    anyMacro,
+    match,
+} from "@unified-latex/unified-latex-util-match";
+import { printRaw } from "@unified-latex/unified-latex-util-print-raw";
+import { replaceNode } from "@unified-latex/unified-latex-util-replace";
+import {
+    splitOnMacro,
+    unsplitOnMacro,
+} from "@unified-latex/unified-latex-util-split";
+import { visit } from "@unified-latex/unified-latex-util-visit";
+
+// all divisions [division macro, environment]
```

Comment:
Make a docstring style comment
```
/**
 * ...
 */
```

Also include in the comment that the order is important and that these are ordered "largest" division to most specialized division.